Lib/ctypes/util.py

```diff
@@ -368,8 +414,8 @@ def test():
             print(f"crypto\t:: {find_library('crypto')}")
             print(f"crypto\t:: {cdll.LoadLibrary(find_library('crypto'))}")
         else:
-            print(cdll.LoadLibrary("libm.so"))
-            print(cdll.LoadLibrary("libcrypt.so"))
+            print(cdll.LoadLibrary(find_library("c")))
+            print(cdll.LoadLibrary("libc.so"))
```

Comment:
Aren’t these two lines doing the same thing now?

Reply:
They should be. IIRC I added those tests to verify that they actually are the same thing.

Reply:
I don't think that would be necessary. The updated test should be good enough for both.

Reply:
I’m unsure about this change.  Maybe the author really wanted to see if libm and libcrypt could be found, not just libc which should always be there.

Reply:
What would be the purpose of testing that? Those libs does not not exist on musl libc systems.

I thought that `libm.so` and `libcrypt.so` were used  because they were expected to always be there. But there is no guarantee that they are.

Reply:
Let me put it this way: should the test fail if `libm.so` or `libcrypt.so` intentionally does not exist on the filesystem?

Reply:
My point was about not changing this test for glibc-based systems – new test for musl can use the two new lines